charts/kubernetes-external-secrets/values.yaml

```diff
@@ -62,8 +62,8 @@ fullnameOverride: ""
 podAnnotations: {}
 podLabels: {}
 
-securityContext: {}
-  # fsGroup: 65534
```

Comment:
Perhaps we can keep this, seems like it's useful for people running IRSA, tho I guess it's not really helpful without any introduction anyway 😅

Reply:
Thanks, good observation; I've added back with a comment.